.github/workflows/make_bundle_conda.yml

```diff
@@ -277,11 +279,18 @@ jobs:
           echo "CONSTRUCTOR_NOTARIZATION_IDENTITY=${notarization_identity}" >> $GITHUB_ENV
 
       - name: Load signing certificate (Windows)
-        if: runner.os == 'Windows'
+        if: runner.os == 'Windows' && github.event_name == 'schedule'
+        # We are signing with Apple's certificate to provide _something_
+        # This is not trusted by Windows so the warnings are still there, but curious users
+        # will be able to check it's actually us if necessary
         run: |
           Set-Content -Path "${{ runner.temp }}/certificate.b64.txt" -Value '${{ secrets.APPLE_APPLICATION_CERTIFICATE_BASE64 }}'
           certutil -decode "${{ runner.temp }}/certificate.b64.txt" "${{ runner.temp }}/certificate.pfx"
 
+          echo "CONSTRUCTOR_SIGNING_CERTIFICATE=${{ runner.temp }}/certificate.pfx" >> $Env:GITHUB_ENV
```

Comment:
the env configuration does not work for windows?

Reply:
It does, but the presence of the keys changes behavior in the script, so we should only set these variables if scheduled, hence the change.